DSCResources/MSFT_xDatabase/MSFT_xDatabase.psm1

```diff
@@ -252,26 +272,119 @@ function DeployDac([string] $databaseName, [string]$connectionString, [string]$s
         $defaultDacPacApplicationVersion = $defaultDacPacApplicationVersion
     }
 
+    Write-Verbose "DeployDac: Loading DacFx"
+
     Load-DacFx -sqlserverVersion $sqlserverVersion
 
-    $dacServicesObject = new-object Microsoft.SqlServer.Dac.DacServices ($connectionString)
+    Write-Verbose "DeployDac: Loading DacPac"
 
-    $dacpacInstance = [Microsoft.SqlServer.Dac.DacPackage]::Load($dacpacPath)
+    try
+    {        
+        $dacpacInstance = Load-Dacpac $path
+    }
+    catch
+    {
+        Write-Verbose("Unable to load dacpac, error: $_")  
```

Comment:
Do you mean Write-Error instead of Write-Verbose? I guess there are a few places where that should be, let me know if that is what you mean and I will go through the module and change them all.